plugins/xpressive/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+INCLUDE(BuildPlugin)
+
+SET(CMAKE_CXX_FLAGS "-Dexprtk_disable_sc_andor -Dexprtk_disable_enhanced_features -Dexprtk_disable_break_continue -Dexprtk_disable_comments -Dexprtk_disable_string_capabilities -Dexprtk_disable_rtl_io_file -Dexprtk_disable_rtl_vecops ${WERROR_FLAGS}")
```

Comment:
Please include previous value of `CMAKE_CXX_FLAGS`. You should apply all these definitions only for the MinGW-w64 build, as recommended at https://github.com/LMMS/lmms/pull/3259#issuecomment-273447592.